src/Collector/Finder/TemplatePathFinder.php

```diff
@@ -28,7 +30,17 @@ public function __construct(CollectedDataNode $collectedDataNode)
             if (!isset($this->collectedTemplatePaths[$className][$methodName])) {
                 $this->collectedTemplatePaths[$className][$methodName] = [];
             }
-            $this->collectedTemplatePaths[$className][$methodName][] = $collectedTemplatePath->getTemplatePath();
+            $templatePath = $collectedTemplatePath->getTemplatePath();
+            if (strpos($templatePath, '*') !== false) {
+                $dirWithoutWildcards = (string)Strings::before((string)Strings::before($templatePath, '*'), '/', -1);
+                $pattern = substr($templatePath, strlen($dirWithoutWildcards) + 1);
+                /** @var string $file */
```

Comment:
why are you adding this php doc and then cast $file to string? casting is just good enough isn't it?

Reply:
PHPStan complained becuse it does not know type of $file